File: november_and_december_2022/SortingtheSentence.java
class Solution {
    public String sortSentence(String s) {
        String[] sortedList = s.split(" ");
            List<String> sorted = new ArrayList<>();
            for (int i = 1; i < sortedList.length + 1; i++){
                for (String word : sortedList){
                    String index = "" + i;
                    if (word.contains(index)){
                        sorted.add(word.replaceAll(index, " "));
                    }
                }
            }
            String result = "";
            for (String word : sorted){
                result += word;
            }
            return result.trim();
        
    }
}

File: november_and_december_2022/SortingtheSentence.py
class Solution:
    def sortSentence(self, s: str) -> str:
        sortedList = []
        words = s.split(" ")
        for i in range(1, len(words) + 1):
            for word in words:
                if str(i) in word:
                    sortedList.append(word.replace(str(i), ""))
        return " ".join(word for word in sortedList)
